{"name": "test_login_propio", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'driver'", "trace": "setup_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"3e8e9a356ed40bf352d1dd1366b4906e\")>\n\n    @allure.feature('Login Tests')\n    def test_login_propio(setup_browser):\n        driver = setup_browser\n    \n        try:\n    \n>           FuncionesPropias.inicioSesion(driver)\n\ntest_3_CierreDeSesion.py:56: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"3e8e9a356ed40bf352d1dd1366b4906e\")>, usuario = 'pormero@colegium.com', clave = 'Pop23725'\n\n    def inicioSesion(self, usuario=usuario, clave=clave):\n        print(\"■■■■■-Inicio inicio sesion-■■■■■\")\n>       self.driver.implicitly_wait(20)\nE       AttributeError: 'WebDriver' object has no attribute 'driver'\n\n..\\Funciones\\FuncionesPropias.py:41: AttributeError"}, "attachments": [{"name": "screenshot_error", "source": "21e157e2-472b-4c91-baaa-2fb8b0dc2216-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "5930bc0b-c8b7-456e-85ce-bc4f831ff0b3-attachment.txt", "type": "text/plain"}], "start": 1723753542970, "stop": 1723753543170, "uuid": "1767f2ba-bc1f-4f7d-95c3-19a21a67c942", "historyId": "16640ab113bed520f58f5a95743c105e", "testCaseId": "16640ab113bed520f58f5a95743c105e", "fullName": "test_3_CierreDeSesion#test_login_propio", "labels": [{"name": "feature", "value": "Login Tests"}, {"name": "suite", "value": "test_3_CierreDeSesion"}, {"name": "host", "value": "DESKTOP-6A7N4QP"}, {"name": "thread", "value": "28660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_3_CierreDeSesion"}]}